hack/fetch-releases.sh

```diff
@@ -120,7 +120,48 @@ release_yaml_pac() {
     echo ""
 }
 
-#Args: <target-platform> <pipelines version> <triggers version> <dashboard version> <results version> <pac version>
+
+release_yaml_hub() {
+  echo fetching '|' component: ${1} '|' version: ${2}
+  local version=$2
+
+  ko_data=${SCRIPT_DIR}/cmd/${TARGET}/operator/kodata
+  dirPath=${ko_data}/tekton-hub/${version}
+  rm -rf ${dirPath} || true
+  mkdir -p ${dirPath} || true
+
+  url=""
+  components="db db-migration"
+
+  if [[ ${TARGET} == "openshift" ]]; then
+    components="$components api-openshift"
+  else
+    components="$components api-k8s"
+  fi
```

Comment:
```suggestion
  components="db db-migration api"
```

is the `if` condition necessary, as this is happening already inside `cmd/<platform>/operator/kodata`

Reply:
oh i see.. 🧑‍💻 
resolving 🤓